middlewares/accessRouteGuard.js

```diff
@@ -0,0 +1,21 @@
+
+const jwt = require("../utils/jwt");
+
+
+
+
+const accesRouteGuard =  (accessRole) => {
+    return async (req,res,next) => {
+    const token = req.headers['x-authorization'];
+    if(!token) {{return res.status(401).json({ message: 'Unauthorized access' });}};
```

Comment:
No need for double {} brackets here. You can even leave out {} and rewrite it as:
`if(!token) res.status(401).json({ message: 'Unauthorized access' })`